transit_odp/organisation/querysets.py

```diff
@@ -1279,3 +1279,7 @@ def add_registration_number(self) -> TSeasonalServiceQuerySet:
                 output_field=CharField(),
             )
         )
+
+    def get_seasonal_service_counter(self, org_id: int) -> int:
```

Comment:
This isn't a very descriptive name for a method on the `SeasonalService` queryset. Better would be something like `get_count_in_organisation`.